apostrophecms/seo
# Implement addition of robots.txt to site

## Summary

*Summarize the changes briefly, including which issue/ticket this resolves. If it closes an existing Github issue, include "Closes #[issue number]"*
This PR adds new functionality to the `seo-fields-global` module. It allows users to set the robots.txt file for the site. Closes PRO-6658. 
## What are the specific steps to test this change?

*For example:*
> 1. Run the website and log in as an admin
> 2. Open a piece manager modal and select several pieces
> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
> 4. Check that all pieces have been archived properly

1. Install the extension
2. Open the global config and select the SEO tab
3. Select to allow indexing and check `http://localhost:3000/robots.txt`
4. Check the other two settings for appropriate robots.txt routes 

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
